scripts/utils.js

```diff
@@ -0,0 +1,70 @@
+// Parse values fetched from Figma by populating designTokens with css variables
+// each variable is associated with all the available themes and each theme is associated with a color defined in the ColorLibrary
+function generateDesignTokens(tokens, theme, semanticColorsLibrary, designTokens) {
+  const themeClass = theme.split('-').pop().toLowerCase();
+
+  for (const token in tokens) {
+    const tokenName = tokens[token].name.split('.');
+    const type = tokenName[0];
+    const name = tokenName.pop();
+
+    if (type !== 'ide-semantic') {
+      if (!designTokens['--arduino-' + name]) {

```

Comment:
You could use string interpolation here `` `--arduino-${name}` `` and maybe you can also define it once a reuse it later.

Reply:
Thank you for the suggestion. I did a little rework applying this too, here: 9bd0001a7083cbfcdd798022412347dfe30f7d53